carma_wm/include/carma_wm/WorldModel.h

```diff
@@ -239,8 +239,8 @@ class WorldModel
    *
    * \return A list of continuous centerline segments and their respective curvatures
    */
-  virtual std::vector<std::tuple<size_t, std::vector<double>>>
-  getLocalCurvatures(const std::vector<lanelet::ConstLanelet>& lanelets) const = 0;
+  static std::vector<double>
+  getLocalCurvatures(const std::vector<lanelet::ConstLanelet>& lanelets);
```

Comment:
Looks like you update its prarm. Can you update the function comments as well?

Reply:
Yeah, will do!